shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2489,7 +2489,13 @@ double WebContents::GetZoomLevel() const {
   return zoom_controller_->GetZoomLevel();
 }
 
-void WebContents::SetZoomFactor(double factor) {
+void WebContents::SetZoomFactor(gin_helper::ErrorThrower thrower,
+                                double factor) {
+  if (factor == 0.0) {
+    thrower.ThrowError("'zoomFactor' must be a double greater than 0.0");
```

Comment:
Could it technically be negative? Idk what happens if you do that

Reply:
yeah - i guess this could be altered to `<= 0.0`?

Reply:
done!